source/configure/elasticsearch-configuration-settings.rst

```diff
@@ -112,47 +113,48 @@ Server password
 | - **false**: **(Default)** Cluster sniffing is disabled .      |                                                                          |
 +----------------------------------------------------------------+--------------------------------------------------------------------------+
 | Select the **Test Connection** button in the System Console to validate the connection between Mattermost and the Elasticsearch server.   |
 +----------------------------------------------------------------+--------------------------------------------------------------------------+
 
 Bulk indexing
--------------
+~~~~~~~~~~~~~
 
 |enterprise| |self-hosted|
 
 *Available in legacy Enterprise Edition E10/E20*
 
 +---------------------------------------------------------------+--------------------------------------------------------------------------+
-| Start a bulk index of all existing posts in the database.     | - System Config path: **Environment > Elasticsearch**                    |
-|                                                               | - ``config.json`` setting: N/A                                           |
+| Configure Mattermost to start a bulk index of all existing    | - System Config path: **Environment > Elasticsearch**                    |
+| posts in the database.                                        | - ``config.json`` setting: N/A                                           |
 |                                                               | - Environment variable: N/A                                              |
 +---------------------------------------------------------------+--------------------------------------------------------------------------+
 | Select the **Index Now** button in the System Console to start a bulk index of all posts. If the indexing process is canceled, the       |
 | index and search results will be incomplete.                                                                                             |
 +---------------------------------------------------------------+--------------------------------------------------------------------------+
 
 Purge indexes
--------------
+~~~~~~~~~~~~~
 
 |enterprise| |self-hosted|
 
 *Available in legacy Enterprise Edition E10/E20*
 
 +---------------------------------------------------------------+-------------------------------------------------------------+
-| Purge the entire Elasticsearch index.                         | - System Config path: **Environment > Elasticsearch**       |        
+| Configure Mattermost to purge the entire Elasticsearch index. | - System Config path: **Environment > Elasticsearch**       |        
 | Typically only used if the index has corrupted and search     | - ``config.json`` setting: N/A                              |
 | isn't behaving as expected.                                   | - Environment variable: N/A                                 |          
```

Comment:
Propose using "functioning" rather than "behaving" as it may not translate well.